resources/QuickSearch.py

```diff
@@ -1,30 +1,68 @@
-from werkzeug.security import generate_password_hash, check_password_hash
 from flask_restful import Resource
-from flask import request, jsonify, make_response
 from middleware.security import api_required
+from utilities.convert_dates_to_strings import convert_dates_to_strings
+import spacy
 
 class QuickSearch(Resource):
   def __init__(self, **kwargs):
     self.supabase = kwargs['supabase']
```

Comment:
can remove

Reply:
Which line are you asking to remove? I think both might still be necessary for this endpoint. The convert_dates_to_strings is needed because the dates returned by Supabase are a type that can't be sent in a response to the client (I was getting a "not JSON serializable" error). The other import, spaCy, is to "depluralize" search terms so that if a user writes "reviews" but the data source is "Allegheny County Police Review Board Transcripts", spaCy will turn it to "review" allowing it to capture this data source that it otherwise wouldn't have

Reply:
Immediately after adding this comment, I realized you're probably talking about the supabase import here haha my bad. I can remove that